src/nvidia/devcontainer-feature.json

```diff
@@ -0,0 +1,17 @@
+{
+  "id": "nvidia",
```

Comment:
Yeah, +1 on this being a gap we should address. This feature won't work at all without some way to set that flag.

Reply:
The workaround today for end users would be to add the following to their devcontainer:

```
"runArgs": ["--gpus", "all"],
"features": {
    "ghcr.io/devcontainers/features/nvidia-cuda:1": {}
}
```

Ideally the feature itself could contribute the `--gpus all` run args so that the end user doesn't need to include it in the devcontainer.

Reply:
Raised https://github.com/devcontainers/spec/issues/82